src/core/components/object-model.jsx

```diff
@@ -194,6 +195,22 @@ export default class ObjectModel extends Component {
                     </td>
                   </tr>
               }
+              {
+                !allOf ? null
+                  : <tr>
+                    <td>{ "allOf ->" }</td>
+                    <td>
+                      {allOf.map((schema, k) => {
```

Comment:
This can fail if `allOf` will be defined as anything else than array. But seeing the `anyOf` and `oneOf`, let's accept that just for consistency. We can fix that in a follow up PR for every `*Of` keyword.